org.goldcoinproject.goldcoin-qt.json

```diff
@@ -0,0 +1,71 @@
+{
+  "id": "org.goldcoinproject.goldcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "22.08",
+  "command": "goldcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "goldcoin",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/goldcoin-qt"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
```

Comment:
I will repeat one more time. This doesn't do what you think it does. 

See this comment 

https://github.com/flathub/flathub/pull/3720#discussion_r1051319764

Reply:
Are you suggesting me that i should remove the **only-arches** directive?

Reply:
did you even read the documentatin I pointed at ?

Reply:
Yes I read it and choose my app build on only x86_64. What's wrong with that?

Reply:
Note this has to be a separate file, as described at https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64. Defining it in the manifest only limits specific module or source, but doesn't limit the build.

Reply:
> Note this has to be a separate file, as described at https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64. Defining it in the manifest only limits specific module or source, but doesn't limit the build.

I have added a file flathub.json and moved relevant lines into it.